src/components/calculator.js

```diff
@@ -11,280 +11,265 @@ function isNumber(item) {
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="8"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="9"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td className="end">
+                {' '}
+                <Button
+                  text="x"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                {' '}
+                <Button
+                  text="4"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="5"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="6"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td className="end">
+                {' '}
+                <Button
+                  text="-"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                {' '}
+                <Button
+                  text="1"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="2"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td>
+                {' '}
+                <Button
+                  text="3"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
+              </td>
+              <td className="end">
+                {' '}
+                <Button
+                  text="+"
+                  handleClick={(e) => {
+                    handleClick(e.target.textContent);
+                  }}
+                />
```

Comment:
- [ ] Please you have `two addition operators` and `no division operator` in your calculator, please kindly fix that as it is a requirement for this project.